src/vectorchord/usage/range-query.md

```diff
@@ -0,0 +1,31 @@
+# Range Query
+
+To query vectors within a certain distance range, you can use the following syntax.
+```SQL
+-- Query vectors within a certain distance range
+SELECT * FROM items WHERE embedding <<->> sphere('[0.24, 0.24, 0.24]'::vector, 0.012);
+```
+
+The expression `sphere('[0.24, 0.24, 0.24]'::vector, 0.012)` refers to a spherical region with its center at `'[0.24, 0.24, 0.24]'` and a radius of `0.012`.
+
+The expression `embedding <<->> sphere('[0.24, 0.24, 0.24]'::vector, 0.012)` evaluates to `true` if the embedding is within the spherical region. Semantically, this is equivalent to `embedding <-> '[0.24, 0.24, 0.24]' < 0.012`.
+
+However, if you have created the index,
+
+<code lang="sql">
```

Comment:
The reason why there a HTML code block is that Github does not support code blocks in suggestions.

Reply:
fixed.